src/fsharp/symbols/Symbols.fsi

```diff
@@ -836,6 +839,9 @@ and [<Class>] public FSharpMemberOrFunctionOrValue =
     /// Get the name as presented in F# error messages and documentation
     member DisplayName : string
 
+    /// List of list of parameters. Typically, there is only one nested list.
```

Comment:
Recommend using a `<summary>` and `<remarks>` now our tooling helps check API docs.  But the rest of the API doesn't use this yet so don't worry too much about it, we could do it in a separate pass.